index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Tic Tac Toe</title>
+    <script src='script.js'></script>
+    <link href="style.css" rel="stylesheet">
+  </head>
+  <body onload="startGame();">
+    <h1>Tic Tac Toe</h1>
+    <div id="message">messages will go here</div>
+    <table border='1'>
+      <tr>
+        <td id='s1' class="square" onClick="nextMove(this);"></td>
+        <td id='s2' class="square" onClick="nextMove(this);"></td>
+        <td id='s3' class="square" onClick="nextMove(this);"></td>
+      </tr>
+      <tr>
+        <td id='s4' class="square" onClick="nextMove(this);"></td>
+        <td id='s5' class="square" onClick="nextMove(this);"></td>
+        <td id='s6' class="square" onClick="nextMove(this);"></td>
+      </tr>
+       <tr>
+         <td id='s7' class="square" onClick="nextMove(this);"></td>
+         <td id='s8' class="square" onClick="nextMove(this);"></td>
+         <td id='s9' class="square" onClick="nextMove(this);"></td>
+      </tr>
+    </table>
+    <a href="javascript:startGame();">Start Over</a>
```

Comment:
I see alot of behavior inside the HTML. I would encourage you to keep the content in HTML and keep the behavior in your JS.